Source/Core/Common/FixedSizeQueue.h

```diff
@@ -21,7 +21,13 @@ class FixedSizeQueue
   void clear()
   {
     if constexpr (!std::is_trivial_v<T>)
-      storage = {};
+    {
+      // The clear of non-trivial objects previously used "storage = {}". However, this causes GCC
+      // to take a very long time to compile the file/function, as well as generating huge amounts
+      // of debug information (~2GB object file, ~600MB of debug info).
+      while (count > 0)
```

Comment:
```suggestion
      storage.fill({});
```

I played around with Compiler Explorer and this is semantically equivalent to initializing with an empty initializer list and produces sane code.

Reply:
Using `.fill` would be nicer if it does work around the bug.

Reply:
As I mentioned on IRC, filling would have the effect of always default constructing N (5000 in the log case) objects vs `count` objects. For the clear button this isn't a big deal but it might matter with other users of the class.

Reply:
In this case I'm cool with it.